apis/quay/v1/quayregistry_types.go

```diff
@@ -30,12 +30,18 @@ import (
 	quaycontext "github.com/quay/quay-operator/pkg/context"
 )
 
+// QuayVersion represents a quay version as a string. Normally this is set using semantic
+// versioning, e.g. 3.7.0.
```

Comment:
is it a build-time constant?

Reply:
nope, the value is set at runtime (env var)